src/engine/bpmcontrol.cpp

```diff
@@ -739,7 +739,7 @@ void BpmControl::slotAdjustRateSlider() {
 void BpmControl::trackLoaded(TrackPointer pNewTrack, TrackPointer pOldTrack) {
     Q_UNUSED(pOldTrack);
     if (m_pTrack) {
-        disconnect(m_pTrack.data(), SIGNAL(beatsUpdated()),
+        disconnect(&*m_pTrack, SIGNAL(beatsUpdated()),
```

Comment:
Note, If we check always for nullptr, we have no issue, but there would be a difference if not.
If m_pTrack is nullptr, get() will return nullptr while the behavior of &* is undefined. However it is practical nullptr according to this: 
http://stackoverflow.com/questions/2727834/c-standard-dereferencing-null-pointer-to-get-a-reference